docs/source/daml/intro/11_StdLib.rst

```diff
@@ -79,7 +79,7 @@ The ``Optional`` type represents a value that may be missing. It's the closest t
     // Do something else
   }
 
-In Daml the same thing would be expressed as
+In Daml the same thing would be expressed as:
```

Comment:
Can you double check this colon and others in the page? I suspect that they fly through RST unchecked but they don't actually render as you would expect. IIRC you need to write `::` to render a single colon.